ivy/container/experimental/layers.py

```diff
@@ -942,3 +942,143 @@ def dct(
             norm=norm,
             out=out,
         )
+
+    @staticmethod
+    def static_stft(
+            signal: Union[ivy.Array, ivy.NativeArray, ivy.Container],
+            /,
+            *,
+            frame_length: Union[int, ivy.Array, ivy.NativeArray, ivy.Container],
+            frame_step: Union[ivy.Array, ivy.NativeArray, ivy.Container],
+            fft_length: Union[ivy.Array, ivy.NativeArray, ivy.Container] = None,
+            window_fn: str = "hann_window",
+            pad_end: bool = False,
+            name: str = None,
+            key_chains: Optional[Union[List[str], Dict[str, str]]] = None,
+            to_apply: bool = True,
+            prune_unapplied: bool = False,
+            map_sequences: bool = False,
+            out: Optional[ivy.Container] = None
+    ) -> ivy.Container:
+        """
+        ivy.Container static method variant of ivy.stft. This method simply
+        wraps the function, and so the docstring for ivy.stft also applies
+        to this method with minimal changes.
+
+        Parameters
+        ----------
+        signal
+            Container with input signal to be transformed
+        frame_length
+            Window length in samples
+        frame_step
+            Number of samples to step
+        fft_length
+            Size of FFT to apply. If not specified, uses the smallest power of 2 closest to frame_length
+        window_fn
+            Window function to use - takes in window length and dtype, returns a tensor of [window_length] samples. No windowing is applied if not specified
+        pad_end
+            Whether to pad the end of signals with zeros when the provided frame length and step produces a frame that lies partially past its end
+        name
+            An optional name for the operation.
+        out
+            Optional output container for writing the results to
+
+        Returns
+        -------
+        ret
+            Container with transformed signals
+
+        """
```

Comment:
One small addition. Could you kindly also add a docstring example for `static_stft`. You may reuse the example you used in the `stft` function below. The only change would be that instead of calling `x.stft(...)`, you'd now call `ivy.Container.static_stft(...)`.